MediaBrowser.Api/Playback/BaseStreamingService.cs

```diff
@@ -368,25 +348,16 @@ private void OnFfMpegProcessExited(Process process, TranscodingJob job, StreamSt
             Logger.LogDebug("Disposing stream resources");
             state.Dispose();
 
-            try
+            if (process.ExitCode == 0)
             {
-                Logger.LogInformation("FFMpeg exited with code {0}", process.ExitCode);
+                Logger.LogInformation("FFMpeg exited with code 0");
             }
-            catch
+            else
             {
-                Logger.LogError("FFMpeg exited with an error.");
+                Logger.LogError("FFMpeg exited with code {0}", process.ExitCode);
             }
 
-            // This causes on exited to be called twice:
```

Comment:
what was the issue that was causing double call, and why it is no longer an issue here?

Reply:
The base object has changed from a custom one to the std one and disposing multiple times shouldn't cause an issue.